SpecializedServers/Horizon/SERVER/Medius/MAPS.cs

```diff
@@ -209,6 +213,25 @@ protected virtual void ProcessMediusPluginMessage(BaseMediusPluginMessage messag
 
                 case NetMessageTypeProtocolInfo protocolInfo:
                     {
+                        byte[] availFactions = new byte[] { 0b00000111, 0, 0, 0 }; //= new byte[4];
+
+                        //0b11100000
+                        //availFactions[0] = 0;
+                        //availFactions[1] = 0;
+                        //availFactions[2] = 0;
+                        //availFactions[3] = 31;
+
+                        data.ClientObject.Queue(new NetMAPSHelloMessage()
+                        {
+                            m_success = true,
+                            m_isOnline = true,
+                            m_availableFactions = availFactions
+
+                        });
+
+
```

Comment:
Try to avoid leaving big spaces in the files.

Reply:
For consistency, I am okay with spacing things out, its how I do organize, especially with a unfinished subserver for Medius no one else can use :)  
You can already tell by the amount of commented code from testing, it is nightmare.
Remember, MAG/Socom 4 don't use Medius like we are normally familiar with.

Reply:
If it is important for your workflow then it will be fine, no need to change it if you work with these in mind.